package DatabaseTesting;



import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;
import java.sql.Statement;
import org.testng.Assert;
import org.testng.annotations.Test;

public class user_accountTableUpdatetDataTest {
    
    @Test
    public void testInsertion() {
        String host = "localhost";
        String port = "3306";
        try (Connection con = DriverManager.getConnection("jdbc:mysql://" + host + ":" + port + "/cooking_project", "root", "Maisa82**");
             Statement s = con.createStatement()) {

            // Execute the INSERT statement
            int rowsAffected = s.executeUpdate("UPDATE user_account SET accountType = 'Seller' WHERE userName = 'Ahmad'");

            // Check if insertion was successful
            Assert.assertEquals(rowsAffected, 1, "Data insertion failed.");
        } catch (SQLException e) {
            e.printStackTrace();
            Assert.fail("Exception occurred: " + e.getMessage());
        }
    }
}